concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -215,14 +221,14 @@ def main():
         force_subtask_results(
             timestamp = timestamp_to_isoformat(current_time),
             ack_report_computed_task =ack_report_computed_task(
-                timestamp = timestamp_to_isoformat(current_time - 28801),  # current_time - 08:00:01
-                subtask_id = task_id + '2C',
+                timestamp = timestamp_to_isoformat(current_time - (cluster_consts.subtask_verification_time * 1.4)),
+                subtask_id = subtask_id,
                 task_to_compute = task_to_compute(
-                    timestamp = timestamp_to_isoformat(current_time - 28802),  # current_time - 08:00:02
+                    timestamp = timestamp_to_isoformat(current_time - cluster_consts.subtask_verification_time * 1.5),
                     compute_task_def = compute_task_def(
-                        task_id     = task_id + '2C',
-                        subtask_id  = task_id + '2C',
-                        deadline    = current_time,
+                        task_id     = subtask_id + '2C',
```

Comment:
Please do this only once too, at the beginning of test scenario. Assign this to `task_id` variable an use it everywhere.

Reply:
Done.